### ISSUE TITLE
add fixtures.js in test folder

### ISSUE THREAD

**Issue by ragnarokatz** [Seneca-CDOT/telescope#318]
In a similar format used by `bull`'s own test suites, I would like us to implement our own set of fixtures to be used for testing. They can be easily used to mock things, as well as reduce repetition in the code.

Bull's own testing fixtures: https://github.com/OptimalBits/bull/tree/develop/test/fixtures

In the immediate time frame, I would like the following code from `test/feed-parser.test.js` to be moved into the proposed `test/fixtures.js`:
```
const nock = require('nock');

...

nock('https://test321.blogspot.com')
    .get('/feeds/posts/default/-/open-source')
    .reply(
      200,
      '<?xml version="1.0" encoding="UTF-8" ?><rss version="2.0"><channel><title>W3Schools Home Page</title><link>https://www.w3schools.com</link><description>Free web building tutorials</description><item><title>RSS Tutorial</title><link>https://www.w3schools.com/xml/xml_rss.asp</link><description>New RSS tutorial on W3Schools</description></item><item><title>XML Tutorial</title><link>https://www.w3schools.com/xml</link><description>New XML tutorial on W3Schools</description></item></channel></rss>'
    );

...

nock('https://test321.blogspot.com')
    .get('/feeds/posts/default/-/open-source?alt=rss')
    .reply(
      200,
      '<?xml version="1.0" encoding="UTF-8" ?><rss version="2.0"><channel></channel></rss>'
    );
```

These code appear repeated in the file, and there's potential for them to be used elsewhere as well (in the upcoming `feed-worker.test.js` file)

**Comment by jerryshueh** [comment]
I don't see any self-assignees or PRs on this issue, so I assume it is up for grabs? If so, I'd like to try my hand at this seeing as I haven't had a hand in anything test-related. This might be a good start :smiley:.

**Comment by ragnarokatz** [comment]
@jerryshueh Yes it is up for grabs. Feel free to work on it.

**Comment by humphd** [comment]
Closed in #321
